message MSG_CLIENT_DM_GROUP_DESCRIPTION_UPDATE
{
    optional string token = 1;
    optional string groupName = 2;
    optional string groupDescription = 3;
}

message MSG_DM_CLIENT_GROUP_DESCRIPTION_UPDATE_ACK
{
    optional int32 statusCode = 1;
}